packages/mobile/src/variables/FontColors.ts

```diff
@@ -0,0 +1,4 @@
+export enum FontColors { 
```

Comment:
Wszystkie właściwości związane z fontem wrzuciłbym do jednego pliku

Reply:
Dla mnie te enumy jako osobne pliki są ok

Reply:
Kolor Headers mi nic nie mówi. Nie wiem jaki to kolor, wiem że to kolor użyty w headers ale jak bede chciał tego koloru użyć do czegoś innego np. tekstu w button to utworze drugi BUTTONS='#34241a' czy zmienie nazwe na HEADERS_And_BUTTON?